### PR TITLE
196 as a dev I want to refactor the header component

### PR DESCRIPTION
**Pull request by SamuelPelletierEvraire**
- [x] Must contain the ACIA logo
- [x] Must contain a button to change language
- [x] Must contain a profile button [profile / dashboard page issue](https://github.com/ai-cfia/fertiscan-frontend/issues/283)
- [x] Must contain a button to display the side menu [side menu issue](https://github.com/ai-cfia/fertiscan-frontend/issues/197)

### CODE REVIEW

**Review by k-allagbe — COMMENTED** [review]
Let's work on user pop up in another issue / PR.

**Comment by SamuelPelletierEvraire** [comment]
- test

**Review by k-allagbe — COMMENTED** [review]
Let's deactivate animations (default MUI ones, and custom ones) to keep things sober. I'm referring to stuff like button waves, sliding components...

Animations like loading indicators are necessary however.

**Comment by SamuelPelletierEvraire** [comment]
> Animations like loading indicators are necessary however.
@k-allagbe 
This is gonna be made in another Issue about skeleton loading

**Comment by SamuelPelletierEvraire** [comment]
Create new issue : Create new issue for separate user Popup and the header.: https://github.com/ai-cfia/fertiscan-frontend/issues/298

File: src/app/layout.tsx [hunk 1]
```diff
@@ -1,50 +1,35 @@
...
-  subsets: ["latin"],
-  display: "swap",
-  variable: "--font-roboto",
-});
-
 export default function RootLayout({
   children,
 }: Readonly<{ children: React.ReactNode }>) {
-  const [SideNavOpen, setSideNavOpen] = useState(false);
+  const { sideNavOpen } = useStore();
```
Comment (k-allagbe) [inline src/app/layout.tsx:14]: Let's use the zustand stores sparsely. As a rule of thumb, when the components sharing the state are grouped and hierarchically not too distant, we should use useState and props drilling. Like in this case with Header and SideNav, they are close enough, using useState is fine.

**Comment by ChromaticPanic** [comment]
It's doing a weird horizontal scroll thing on the sidenav

image

**Comment by SamuelPelletierEvraire** [comment]
> It's doing a weird horizontal scroll thing on the sidenav . Left Bottom. Also the vertical scrollbar persists no matter the browser height
> 
> image
> 
> image

The side nav was already implemented but not done at all gonna be done in other issue.

File: src/app/layout.tsx (hunk 1 above)
Comment (SamuelPelletierEvraire) [reply src/app/layout.tsx]: Done

**Review by Youllou — APPROVED** [review]

**Review by k-allagbe — APPROVED** [review]

**Pull request merged**